package.json

```diff
@@ -42,27 +42,34 @@
     "@jsdevtools/karma-config": "^3.1.7",
     "@readme/eslint-config": "^8.7.3",
     "@types/chai": "^4.3.1",
+    "@types/express": "^4.17.13",
     "@types/mocha": "^9.1.1",
+    "@types/multer": "^1.4.7",
     "@types/node": "^18.0.0",
     "@types/parse-data-url": "^3.0.0",
     "@types/readable-stream": "^2.3.13",
     "chai": "^4.3.4",
+    "datauri": "^4.1.0",
     "eslint": "^8.14.0",
+    "express": "^4.18.1",
     "fetch-mock": "^9.11.0",
     "form-data": "^4.0.0",
     "form-data-encoder": "^1.7.1",
     "formdata-node": "^4.3.2",
     "har-examples": "^3.1.1",
     "isomorphic-fetch": "^3.0.0",
     "mocha": "^10.0.0",
+    "multer": "^1.4.5-lts.1",
     "nock": "^13.2.4",
     "node-fetch": "^2.6.0",
     "nyc": "^15.1.0",
     "prettier": "^2.6.2",
-    "ts-loader": "^9.3.0",
+    "temp-dir": "^2.0.0",
+    "ts-loader": "^8.4.0",
```

Comment:
Had to downgrade this and `@jsdevtools/karma-config` because a PR I merged in to upgrade `ts-loader` late last month broke browser tests without me realizing. `@jsdevtools/karma-config` uses Webpack 4 and `ts-loader@9` only supports Webpack 5.